types/src/legacy_sse_data/translate_execution_result.rs

```diff
@@ -201,7 +209,88 @@ mod tests {
             Some(TransformKindV1::WriteContractPackage),
             maybe_tanslate_stored_value(&stored_value)
         );
-        //TODO wrtite tests for rest of cases
```

Comment:
The structures behind new variants in StoredValue don't have `random` implementations, and it's not clear how to construct some of these structs. So I give up having 100% coverage here, focusing on those that can change the output to None.